iati/tests/fixtures/versions.py

```diff
@@ -0,0 +1,428 @@
+"""Pytest fixtures for specifying versions.
+
+---
+
+There is a standard method of defining fixtures that return version numbers. This is:
+
+    `std_ver_[component_level]_[instantiation_status]_[validity]_[type(s)]_[knowledge]`
```

Comment:
I wonder if there is any way of making this naming structure clearer (so that when reading a variable name in the tests it is more obvious what is contains). However I've not yet thought of any practical suggestions to offer.

Reply:
The one large-ish problem with this structure is potentially redundant information in the names.

For example, if there is a fixture that returns `iati.Version`s, it is by definition both instantiated and valid. As such, it is not necessary to provide all this information. It could, however, be useful to explicitly state some things that can be inferred.

There are a couple of the current fixtures where the balance doesn't feel quite right, though I've not been able to think of an alternative that conveys the information in a better manner.

Reply:
Yep see your point - probably not a blocker for this PR though.